config/rbac/role.yaml

```diff
@@ -204,3 +204,29 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
We need to provide `create` access to `all` the resources of group `infrastructure.cluster.x-k8s.io`, just like [MachineController](https://github.com/kubernetes-sigs/cluster-api/blob/main/internal/controllers/machine/machine_controller.go#L72)

Reply:
done